cli/commands/project_sync.go

```diff
@@ -142,30 +168,32 @@ func (cmd *ProjectSync) StartUnisonSync(ctx *cli.Context, volumeName string, con
 		}
 	}
 	cmd.out.Verbose.Printf("Unison Args: %s", strings.Join(unisonArgs[:], " "))
-	if err = exec.Command("unison", unisonArgs...).Start(); err != nil {
+	command := exec.Command("unison", unisonArgs...)
+	// While the logFile is an absolute path to the workingDirectory, the Unison
+	// process is still watching the "current directory" and so needs the working
+	// directory to be explicitly set.
+	command.Dir = workingDir
+	cmd.out.Verbose.Printf("Script execution - Working Directory: %s", workingDir)
```

Comment:
Log message appears wrong. This should be `Sync` or something besides `Script execution`